diff --git a/node_modules/steamapi/dist/src/SteamAPI.js b/node_modules/steamapi/dist/src/SteamAPI.js
index 1a0b64c..8114eda 100644
--- a/node_modules/steamapi/dist/src/SteamAPI.js
+++ b/node_modules/steamapi/dist/src/SteamAPI.js
@@ -3,7 +3,6 @@ import querystring from 'node:querystring';
 // https://stackoverflow.com/a/66726426/7504056
 import { createRequire } from 'node:module';
 const require = createRequire(import.meta.url);
-const Package = require('../../package.json');
 import { MemoryCacheMap } from './Cache.js';
 import { fetch, assertApp, assertID } from './utils.js';
 import UserStats from './structures/UserStats.js';
@@ -24,7 +23,7 @@ import UserSummary from './structures/UserSummary.js';
 const defaultOptions = {
     language: 'english',
     currency: 'us',
-    headers: { 'User-Agent': `SteamAPI/${Package.version} (https://www.npmjs.com/package/${Package.name})` },
+    headers: { 'User-Agent': `SteamAPI/3.0.8 (https://www.npmjs.com/package/steamapi)` },
     baseAPI: 'https://api.steampowered.com',
     baseStore: 'https://store.steampowered.com/api',
     baseActions: 'https://steamcommunity.com/actions',
